### PR TITLE
Headless menu

### PR DESCRIPTION
**Pull request by Obvio**
# What is it?

- [X] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. Menu with buttons to perform actions

# Screenshots/Demo

https://user-images.githubusercontent.com/300232/219660734-51ae1a50-df64-4091-8742-7004aa36ec55.mov

# Checklist:

- [X] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/website/src/components/menu/menu.tsx [hunk 1]
```diff
@@ -22,8 +22,9 @@ export const Menu = component$<Props>(({ onClose$ }) => {
       path: `/docs/${appState.theme.toLowerCase()}/collapse`,
     },
     { label: 'Drawer', path: `/docs/${appState.theme.toLowerCase()}/drawer` },
-    { label: 'Rating', path: `/docs/${appState.theme.toLowerCase()}/rating` },
+    { label: 'Menu', path: `/docs/${appState.theme.toLowerCase()}/menu` },
     { label: 'Popover', path: `/docs/${appState.theme.toLowerCase()}/popover` },
+    { label: 'Rating', path: `/docs/${appState.theme.toLowerCase()}/rating` },
```
Comment (Obvio) [inline packages/website/src/components/menu/menu.tsx:27]: ordered items alphabetically.

**Comment by Obvio** [comment]
@gioboa I moved it from #152, and it includes fixes for your feedback.

**Comment by gioboa** [comment]
Looking good to me! 👏

**Comment by gioboa** [comment]
@all-contributors please add @Obvio for code, ideas, doc, bug and a11y

**Pull request merged**